src/test/steps/loginSteps.ts

```diff
@@ -0,0 +1,35 @@
+import { Given, When, Then } from "@cucumber/cucumber";
+import { chromium, Page, Browser } from "@playwright/test";
+
+let browser: Browser;
+let page: Page;
+
+Given('User navigates to the application', async function () {
+    browser = await chromium.launch({headless: false});
+    page = await browser.newPage();
+    await page.goto("https://px-stage.self-service.danads.com");
+});
+
+Given('User clicks login button', async function () {
+    //await page.locator("//a/*[text()='Log in']").click();
+})
+
+Given('User enters username', async function () {
+    page.locator("(//input[@name='username'])[2]").type("Username");
+});
+
+When('User enters log in with email button', async function () {
+    //page.locator("//span[text()=' Log in with email ']").click();
+});
+
+When('User enters password', async function () {
+    page.locator("(//input[@name='password'])[2]").type("Password");
```

Comment:
It's not a good idea to expose secrets (passwords). Also, the password in Cognito is different.